### ISSUE TITLE
dump_results() got an unexpected keyword argument 'output_format'

### ISSUE THREAD

**Issue by hsp2454** [open-mmlab/mmaction2#2137]
When I run "python tools\test.py configs\skeleton\stgcn\stgcn_80e_ntu60_xsub_keypoint.py work_dirs\stgcn_climb\ lattest.pth --eval top_k_accuracy --cfg-option gpu_ids=[0] ", he reported the following error
![image](https://user-images.githubusercontent.com/115400289/207534612-b4fbc6a7-dbbc-41a8-92f2-fc6464a50f96.png)

Here is my configuration file：
model = dict(
    type='SkeletonGCN',
    backbone=dict(
        type='STGCN',
        in_channels=3,
        edge_importance_weighting=True,
        graph_cfg=dict(layout='coco', strategy='spatial')),
    cls_head=dict(
        type='STGCNHead',
        num_classes=5,
        in_channels=256,
        loss_cls=dict(type='CrossEntropyLoss')),
    train_cfg=None,
    test_cfg=None)
dataset_type = 'PoseDataset'
ann_file_train = 'data\ClimbAcross\coco\\train.pkl'
ann_file_val = 'data\ClimbAcross\coco\\val.pkl'
train_pipeline = [
    dict(type='PaddingWithLoop', clip_len=300),
    dict(type='PoseDecode'),
    dict(type='FormatGCNInput', input_format='NCTVM'),
    dict(type='PoseNormalize'),
    dict(type='Collect', keys=['keypoint', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['keypoint'])
]
val_pipeline = [
    dict(type='PaddingWithLoop', clip_len=300),
    dict(type='PoseDecode'),
    dict(type='FormatGCNInput', input_format='NCTVM'),
    dict(type='PoseNormalize'),
    dict(type='Collect', keys=['keypoint', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['keypoint'])
]
test_pipeline = [
    dict(type='PaddingWithLoop', clip_len=300),
    dict(type='PoseDecode'),
    dict(type='FormatGCNInput', input_format='NCTVM'),
    dict(type='PoseNormalize'),
    dict(type='Collect', keys=['keypoint', 'label'], meta_keys=[]),
    dict(type='ToTensor', keys=['keypoint'])
]
data = dict(
    videos_per_gpu=16,
    workers_per_gpu=2,
    test_dataloader=dict(videos_per_gpu=1),
    train=dict(
        type='PoseDataset',
        ann_file='data\ClimbAcross\coco\train.pkl',
        data_prefix='',
        pipeline=[
            dict(type='PaddingWithLoop', clip_len=300),
            dict(type='PoseDecode'),
            dict(type='FormatGCNInput', input_format='NCTVM'),
            dict(type='PoseNormalize'),
            dict(type='Collect', keys=['keypoint', 'label'], meta_keys=[]),
            dict(type='ToTensor', keys=['keypoint'])
        ]),
    val=dict(
        type='PoseDataset',
        ann_file='data\ClimbAcross\coco\\val.pkl',
        data_prefix='',
        pipeline=[
            dict(type='PaddingWithLoop', clip_len=300),
            dict(type='PoseDecode'),
            dict(type='FormatGCNInput', input_format='NCTVM'),
            dict(type='PoseNormalize'),
            dict(type='Collect', keys=['keypoint', 'label'], meta_keys=[]),
            dict(type='ToTensor', keys=['keypoint'])
        ]),
    test=dict(
        type='PoseDataset',
        ann_file='data\ClimbAcross\coco\\val.pkl',
        data_prefix='',
        pipeline=[
            dict(type='PaddingWithLoop', clip_len=300),
            dict(type='PoseDecode'),
            dict(type='FormatGCNInput', input_format='NCTVM'),
            dict(type='PoseNormalize'),
            dict(type='Collect', keys=['keypoint', 'label'], meta_keys=[]),
            dict(type='ToTensor', keys=['keypoint'])
        ]))
optimizer = dict(
    type='SGD', lr=0.1, momentum=0.9, weight_decay=0.0001, nesterov=True)
optimizer_config = dict(grad_clip=None)
lr_config = dict(
    policy='step', step=[50, 100, 150, 200, 250, 300, 350, 400, 450, 490])
total_epochs = 500
checkpoint_config = dict(interval=5)
evaluation = dict(
    interval=5, metrics=['top_k_accuracy', 'mean_class_accuracy'])
log_config = dict(interval=100, hooks=[dict(type='TextLoggerHook')])
dist_params = dict(backend='nccl')
log_level = 'INFO'
work_dir = './work_dirs/stgcn_climb/'
load_from = None
resume_from = None
workflow = [('train', 1), ('val', 1)]
output_config = dict(
    out='./work_dirs/stgcn_climb//results.json', output_format='json')
gpu_ids = [0]
omnisource = False
module_hooks = []

**Comment by hsp2454** [comment]
@cir7 Hello, I tried to run with the following command, it still reported an error, I don't understand how to remove output_format argument, or just use --out to specify output result in test.py
![image](https://user-images.githubusercontent.com/115400289/207592144-066884d2-6353-4273-91bd-cdfb20fb1810.png)

**Comment by cir7** [comment]
Hi, @hsp2454 , modify 
`output_config = dict(out='./work_dirs/stgcn_climb//results.json', output_format='json')` to 
`output_config = dict(out='./work_dirs/stgcn_climb//results.json)` in your config file, or remove `output_config` and add `--out` as you already did.

**Comment by hsp2454** [comment]
@cir7 
Ok, thank you. Problem solved.
